src/main/java/step2/WinnerMap.java

```diff
@@ -0,0 +1,27 @@
+package step2;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static step2.LottoMatch.*;
+
+public class WinnerMap {
+
+    private static Map<String, Integer> winnerMap = new HashMap<>();
+
+    static void winnerMapInit() {
+        winnerMap.put(FIFTH.name(), 0);
+        winnerMap.put(FOURTH.name(), 0);
+        winnerMap.put(THIRD.name(), 0);
+        winnerMap.put(SECOND.name(), 0);
+        winnerMap.put(FIRST.name(), 0);
+    }
+
+    public static void plusWinnerMapCount(LottoMatch lottoMatch, int matchCount) {
```

Comment:
matchCount는 사용하고 있지 않네요 😄 
```suggestion
    public static void plusWinnerMapCount(LottoMatch lottoMatch) {
```